src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/MetaKeywords.java

```diff
@@ -122,7 +122,7 @@ public List<String> getMetaKeywordsForModule(ModuleElement mdle) {
      * Get the overview keywords.
      */
     public List<String> getOverviewMetaKeywords(String title, String docTitle) {
-         List<String> result = new ArrayList<>(1);
+        List<String> result = new ArrayList<>(1);
```

Comment:
Consider using `var` similarly to my suggestion above. Alternatively, we can use two calls to `List.of(e)` and one call to `List.of()`.

Reply:
Most of these cases seem better with appropriate variants of `List.of`